src/services/services.ts

```diff
@@ -149,8 +149,8 @@ module ts {
             return getSourceFileOfNode(this);
         }
 
-        public getStart(sourceFile?: SourceFile): number {
-            return getTokenPosOfNode(this, sourceFile);
+        public getStart(sourceFile?: SourceFile, skipDecorators?: boolean): number {

```

Comment:
does this need to be in getStart? i would think of this as a helper that the formatter would call. not everyone would want to do that.